projects/react/src/lib/useMaskito.ts

```diff
@@ -0,0 +1,56 @@
+import {Maskito, MASKITO_DEFAULT_OPTIONS, MaskitoOptions} from '@maskito/core';
+import {RefCallback, useCallback, useState} from 'react';
+
+import {useIsomorphicLayoutEffect} from './useIsomorphicLayoutEffect';
+
+export type MaskitoPredicate = (
+    element: HTMLElement,
+) => HTMLInputElement | HTMLTextAreaElement;
+
+const defaultMaskitoPredicate: MaskitoPredicate = e =>
```

Comment:
This one repeats too, I believe

Reply:
https://github.com/Tinkoff/maskito/pull/273#discussion_r1180076265

Reply:
closing in favour of https://github.com/Tinkoff/maskito/pull/273#discussion_r1180178786